sam-karis/airtech
# Ft email notification 167660077

## What does this PR do?
------
It adds email notification to travelers when their flight is remaining 24 hours and update notification

## Description of Task to be completed?
-----
- add automatic email notifications
- update ticket model
- add email template
- update readme

## How should this be manually tested?
------
clone the repo
check out to `ft-email-notification-167660077`
add environment variables as defined in [`.env_example`](https://github.com/sam-karis/airtech/blob/develop/.env_example)
Create a database and run `python manage.py migrate` to create relevant tables.


## Any background context you want to provide?
------
None

## What are the relevant pivotal tracker stories?
------
[email notification](https://www.pivotaltracker.com/story/show/167660077)


## Screenshots (if appropriate)
------
#### email template
![image](https://user-images.githubusercontent.com/22454909/63560602-4dd6d280-c55f-11e9-841c-7651d98dc659.png)
